tests/module-setup.sh

```diff
@@ -43,7 +43,14 @@ elif [[ $MACHINE_ID = cheyenne* ]] ; then
         source /glade/u/apps/ch/modulefiles/default/localinit/localinit.sh
     fi
     module purge
-
+    
```

Comment:
This is hacky and terrible. Why are testing scripts trying to automatically determine what machine they are being run on instead of just being specified on the command line?

Reply:
I think that this has been around for as long as I've been involved with the GFS. I suppose that it's handy for the end user -- one less thing to specify when running tests.